doc/releases/changelog-0.35.0.md

```diff
@@ -6,20 +6,150 @@
 
 <h4>Easy to import circuits 💾</h4>
 
-* An error message provides clearer instructions for installing PennyLane-Qiskit if the `qml.from_qiskit`
-  function fails because the Qiskit converter is missing.
+* This version of PennyLane makes it easier to import workflows from Qiskit.
+
+  The `qml.from_qiskit` function converts a Qiskit
+  [QuantumCircuit](https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.QuantumCircuit) into
+  a PennyLane
+  [quantum function](https://docs.pennylane.ai/en/stable/introduction/circuits.html#quantum-functions).
+  Although `qml.from_qiskit` already exists in PennyLane, we have made a number of improvements to
+  make importing from Qiskit easier:
   [(#5218)](https://github.com/PennyLaneAI/pennylane/pull/5218)
+  [(#5168)](https://github.com/PennyLaneAI/pennylane/pull/5168)
+
+  * You can now append PennyLane measurements onto the quantum function returned by
+    `qml.from_qiskit`. Consider this simple Qiskit circuit:
+
+    ```python
+    import pennylane as qml
+    from qiskit import QuantumCircuit
+
+    qc = QuantumCircuit(2)
+    qc.rx(0.785, 0)
+    qc.ry(1.57, 1)
+    ```
+    
+    We can convert into a PennyLane QNode in just a few lines.
+
+    ```pycon
+    >>> dev = qml.device("default.qubit")
+    >>> measurements = [qml.expval(qml.Z(0) @ qml.Z(1))]
```

Comment:
TODO: pop this into a single expectation value
@Mandrenkov

Reply:
@trbromley See #5268 and https://github.com/PennyLaneAI/pennylane-qiskit/pull/466.

Reply:
@isaacdevlugt Make sure you're using `pennylane-qiskit@master` too!

Reply:
Thanks @Mandrenkov, confirmed that it works for me!

> Also might be worth choosing a different measurement since the Pauli op renaming feature hasn't been shown yet 😅

It's a good point, but I feel like changing it up to something like `Hadamard` feels more distracting. Any recommendations, would you do Hadamards here?

Reply:
```suggestion
    >>> measurements = qml.expval(qml.Z(0) @ qml.Z(1))
```

Reply:
Can we just do `probs`?

Reply:
And thanks @Mandrenkov! Didn't catch that 😅

Reply:
Everything runs fine :)